matrix-sdk-android/src/main/res/values/strings.xml

```diff
@@ -2,53 +2,85 @@
 <resources>
     <string name="summary_message">%1$s: %2$s</string>
     <string name="summary_user_sent_image">%1$s sent an image.</string>
+    <string name="summary_you_sent_image">You sent an image.</string>
     <string name="summary_user_sent_sticker">%1$s sent a sticker.</string>
+    <string name="summary_you_sent_sticker">You sent a sticker.</string>
 
     <string name="notice_room_invite_no_invitee">%s\'s invitation</string>
+    <string name="notice_room_invite_no_invitee_by_you">Your invitation</string>
     <string name="notice_room_created">%1$s created the room</string>
+    <string name="notice_room_created_by_you">You created the room</string>
     <string name="notice_room_invite">%1$s invited %2$s</string>
+    <string name="notice_room_invite_by_you">You invited %1$s</string>
     <string name="notice_room_invite_you">%1$s invited you</string>
     <string name="notice_room_join">%1$s joined the room</string>
+    <string name="notice_room_join_by_you">You joined the room</string>
     <string name="notice_room_leave">%1$s left the room</string>
+    <string name="notice_room_leave_by_you">You left the room</string>
     <string name="notice_room_reject">%1$s rejected the invitation</string>
+    <string name="notice_room_reject_by_you">You rejected the invitation</string>
     <string name="notice_room_kick">%1$s kicked %2$s</string>
+    <string name="notice_room_kick_by_you">You kicked %1$s</string>
     <string name="notice_room_unban">%1$s unbanned %2$s</string>
+    <string name="notice_room_unban_by_you">You unbanned %1$s</string>
     <string name="notice_room_ban">%1$s banned %2$s</string>
+    <string name="notice_room_ban_by_you">You banned %1$s</string>
     <string name="notice_room_withdraw">%1$s withdrew %2$s\'s invitation</string>
+    <string name="notice_room_withdraw_by_you">You withdrew %1$s\'s invitation</string>
     <string name="notice_avatar_url_changed">%1$s changed their avatar</string>
+    <string name="notice_avatar_url_changed_by_you">You changed your avatar</string>
     <string name="notice_display_name_set">%1$s set their display name to %2$s</string>
+    <string name="notice_display_name_set_by_you">You set your display name to %1$s</string>
     <string name="notice_display_name_changed_from">%1$s changed their display name from %2$s to %3$s</string>
+    <string name="notice_display_name_changed_from_by_you">You changed their display name from %1$s to %2$s</string>
```

Comment:
Should be:
You changed **your** display name from %1$s to %2$s

Reply:
good catch, thanks!